Makefile

```diff
@@ -37,4 +37,8 @@ test:
 clean:
 	$(RM) $(EXEC) $(OBJS) $(deps)
 
+.PHONY: regen-opcode
+regen-opcode:
```

Comment:
The target in Makefile should be `opcode.h`, generated by `scripts/gen_opcode.py`.